package.json

```diff
@@ -4,6 +4,11 @@
   "description": "An item manager for Destiny.",
   "main": "app/index.html",
   "scripts": {
+    "dev-chrome": "./node_modules/.bin/grunt dev-chrome",
+    "dev-firefox": "./node_modules/.bin/grunt dev-firefox",
+    "generate-css": "./node_modules/.bin/grunt css",
```

Comment:
@bhollis Good catch. Will update the README. Sorry about that.

And to answer your question, that's up to you? My goal was to simplify the amount of global stuff one needed to install and reference in order to get this running locally (as well as simplify the set up instructions in the README). This takes the readme instructions down from 3 different build tools to only one and doesn't require the user to concern themselves with what tool does what. That's abstracted into npm scripts. 

If you'd like to add (or would like me to add) more npm tasks that mirror grunt tasks that can definitely be done.